pyro/poutine/trace_messenger.py

```diff
@@ -41,7 +41,7 @@ class TraceMessenger(Messenger):
     We can also use this for visualization.
     """
 
-    def __init__(self, graph_type=None, param_only=None):
+    def __init__(self, graph_type=None, param_only=None, strict=None):
```

Comment:
Do you foresee any future notions of strictness you'd like to add? If so, consider renaming this kwarg to `strict_names`.

Reply:
done